app/assets/javascripts/trim_link.js

```diff
@@ -1,71 +1,72 @@
 // The functions in this file manages all the behaviour uploads / cancels to trim.
 // in both the new and in progress tables on the dashboard
 
+// External
+var $;
 
-var trim_link = {};
+var trimLink = {};
 
 // Prepare the callback on trim upload controls
-trim_link.trimFileUpload = function() {
+trimLink.trimFileUpload = function() {
   'use strict';
-  $('.trim-links-form').each(function(i, area){
-    var container = $(area),
-      message_container = container.find('.tr5-message'),
-      message_icon = message_container.find('.fa'),
-      upload_message = message_icon.siblings('.message'),
-      actions = container.find('.tr5-actions'),
-      choose_button = container.find('.button-choose'),
-      form = container.find('form'),
-      file_field = form.find('.trim-file-chooser'),
-      cancel_button = container.find('.button-cancel'),
-      status_messages = {
-        selected : {
-          message : 'File selected',
-          classname : 'fa fa-check-circle'
-        },
-        uploading : {
-          message : 'Uploading',
-          classname : 'fa fa-spin fa-circle-o-notch'
-        },
-        success : {
-          classname : 'fa fa-check-circle'
-        },
+  $('.trim-links-form').each(function(i, area) {
+    var container = $(area);
+    var messageContainer = container.find('.tr5-message');
+    var messageIcon = messageContainer.find('.fa');
+    var uploadMessage = messageIcon.siblings('.message');
+    var actions = container.find('.tr5-actions');
+    var chooseButton = container.find('.button-choose');
+    var form = container.find('form');
+    var fileField = form.find('.trim-file-chooser');
+    var cancelButton = container.find('.button-cancel');
+    var statusMessages = {
```

Comment:
Should "the above block of variables "  be prefixed with a $ as in other examples?